extensions/parquet/compression/src/main/java/io/deephaven/parquet/compress/LZ4WithLZ4RawBackupCompressorAdapter.java

```diff
@@ -51,17 +55,22 @@ public BytesInput decompress(final InputStream inputStream, final int compressed
         // Buffer input data in case we need to retry with LZ4_RAW.
         final BufferedInputStream bufferedInputStream = new BufferedInputStream(inputStream, compressedSize);
```

Comment:
We shouldn't need to create a new buffered input stream if `inputStream.markSupported() == true` IMO.

Reply:
You are right. The idea here though was that we should buffer the bytes to avoid reading again. That is why I used a BufferedInputStream here.

Reply:
Usually a stream that supports marked is already buffered. Additionally, it looks like the implementation buffers internally later if necessary (`org.apache.commons.io.IOUtils#buffer(java.io.InputStream)`)?